specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-04-01-preview/connectedClusters.json

```diff
@@ -149,6 +149,12 @@
               "$ref": "#/definitions/ConnectedCluster"
             }
           },
+          "202": {
```

Comment:
>202 [](http://example.com/codeflow?start=11&length=3)

This requires x-ms-long-running-operation extension.

Reply:
Added